citrination_client/client.py

```diff
@@ -304,6 +304,18 @@ def upload_file(self, file_path, dataset_id, root_path=None):
         """
         return self.upload(dataset_id, file_path)
 
+    def _upload_put_url_headers(self):
+        """
+        Returns headers necessary for requests to presigned put urls
+        :return: A dictionary representing the headers required for requests
+                to presigned URLs for uploading files
+        """
+        headers = {
+            "x-amz-server-side-encryption": "AES256",
+            "x-amz-acl": "private"
```

Comment:
Is this _strictly necessary_?  I'm not objecting to it's presence, just wondering if it is _necessary_ for the uploads to work.